velox/common/memory/MemoryPool.h

```diff
@@ -320,9 +320,17 @@ class MemoryPool : public std::enable_shared_from_this<MemoryPool> {
   /// 'capacity()' is fixed and set to 'maxCapacity()' on creation.
   virtual int64_t capacity() const = 0;
 
-  /// Returns the currently used memory in bytes of this memory pool.
+  /// Returns the currently used memory in bytes of this memory pool. For
+  /// non-leaf memory pool, the function returns the aggregated memory
+  /// reservation from all its child memory pools.
```

Comment:
Is it possible that we remove this currentBytes() API as it can be replaced with either usedBytes or reservedBytes in different scenarios? This API seems to be somewhat confusing.

Reply:
This needs to be careful as we are using currentBytes in many places in production code.